.gitignore

```diff
@@ -2,3 +2,6 @@
 build
 node_modules
 madlib_modules
+.tests
+.coverage
+.obsidian
```

Comment:
You probably should create a `.madlibignore` with that one in there as well. Otherwise it's used by the `madlib package` command to compute the hash.